daemon/errors.go

```diff
@@ -160,6 +160,17 @@ func setExitCodeFromError(setExitCode func(exitStatus), err error) error {
 		return startInvalidConfigError(errDesc)
 	}
 
+	// Go 1.20 changed the error for attempting to execute a directory from
+	// syscall.EACCESS to syscall.EISDIR. Unfortunately docker/cli checks
+	// whether the error message contains syscall.EACCESS.Error() to
+	// determine whether to exit with code 126 or 125, so we have little
```

Comment:
For reference, [this is the offending docker/cli code.](
https://github.com/docker/cli/blob/e92dd87c3209361f29b692ab4b8f0f9248779297/cli/command/container/run.go#L301-L316)

Reply:
Oh my god, when I glanced at this failure, I thought it might be something like that, but I did not fully comprehend the magnitude of coupling.

Reply:
https://github.com/docker/cli/pull/4001